### PR TITLE
Add energy landingpage

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

- Adds a new "landingpage" layout
- Use this layout for the Blue landingpage
- Add landingpage for energy

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/home-energy-management/index.html [hunk 1]
```diff
@@ -51,15 +57,19 @@
...
-    <a href="/getting-started">
-        <div class="banner-overlay-button bottom right">
-            <div>Get started</div>
-        </div>
-    </a>
+    <div class="links">
+      <a href="/getting-started">
+        Get started
+      </a>
+      <a href="https://demo.home-assistant.io/#/energy">
```
Comment (ludeeus) [inline source/home-energy-management/index.html]:
```suggestion
      <a href="https://demo.home-assistant.io/#/energy" target="_blank">
```
When we navigate out of the site, we should do so in a new tab to not lose the viewer

**Review by ludeeus — COMMENTED** [review]
Can't approve my own PR, but here's a :heavy_check_mark: for the commits by @balloob

File: source/home-energy-management/index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+hero_image: /images/energy/hero.jpg
+og_image: /images/blog/2021-08-energy/social.png
+tagline: The world is heating up. Together we can fix that.
+sections:
+  - title: "Monitor usage"
+    description: "Monitor the energy usage, transition to sustainable energy and save money."
+    image: /images/energy/vector-graph.png
+    entries:
+      - title: "Know where it goes."
+        description: "Home Assistant shows at a glance how much you're consumping, producing and where this energy is going."
```
Comment (frenck) [inline source/home-energy-management/index.html]:
```suggestion
        description: "Home Assistant shows at a glance how much you're consuming, producing and where this energy is going."
```

File: source/home-energy-management/index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+          We created SlimmeLezer, a device that connects to electricity meters with P1 ports and makes the data locally available in Home Assistant.
+        image: /images/docs/energy/slimmelezer.jpg
+        read_more: /docs/energy/electricity-grid/#connect-using-a-p1-port
+      - title: "Home Assistant Glow."
+        description: "We created Home Assistant Glow, a device that reads the consumption LED on your electricity meter and make it locally available in Home Assistant."
+        image: /images/docs/energy/home-assistant-glow.jpg
+        image_description: "Home Assistant Glow attached to an energy meter"
+        read_more: /docs/energy/electricity-grid/#reading-the-meter-via-a-pulse-counter
+
+  - title: "Super charge your solar panels"
```
Comment (frenck) [inline source/home-energy-management/index.html]:
```suggestion
  - title: "Supercharge your solar panels"
```

File: source/home-energy-management/index.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    description: "Schedule power usage based on solar production predictions."
+    image: /images/energy/vector-sun.png
+    entries:
+      - title: "Time your usage."
+        description: "It’s cheaper to consume the energy you produce than to give it back to the grid. Home Assistant helps you turn on the power hungry things when your usage needs a boost."
```
Comment (frenck) [inline source/home-energy-management/index.html]:
```suggestion
        description: "It’s cheaper to consume the energy you produce than to give it back to the grid. Home Assistant helps you turn on the power-hungry things when your usage needs a boost."
```

**Review by frenck — APPROVED** [review]

**Pull request merged**